packages/world/src/World.sol

```diff
@@ -72,12 +82,12 @@ contract World is StoreRead, IStoreData, IWorldKernel {
    * Requires the caller to own the root namespace.
    * The module is delegatecalled and installed in the root namespace.
    */
-  function installRootModule(IModule module, bytes memory args) public {
+  function installRootModule(IModule module, bytes memory args) public requireNoCallback {
     AccessControl.requireOwner(ROOT_NAMESPACE_ID, msg.sender);
     _installRootModule(module, args);
   }
 
-  function _installRootModule(IModule module, bytes memory args) internal {
+  function _installRootModule(IModule module, bytes memory args) internal requireNoCallback {
```

Comment:
given that this is internal, do we need this check?

Reply:
nope, already checked by `installRootModule` and `installModule`, good catch